server/loop.c

```diff
@@ -38,7 +38,7 @@ jd_handle_message (JMessage* message, GSocketConnection* connection, JMemoryChun
 	gchar const* path;
 	guint32 operation_count;
 	JBackendOperation backend_operation;
-	g_autoptr(JSemantics) semantics;
+	g_autoptr(JSemantics) semantics = NULL;
```

Comment:
Probably not strictly necessary since `semantics` is initialized a few lines further down. Doesn't hurt, though.

Reply:
I used "grep" to look for all g_autoptr and g_autofree, and there were just a few missing initializations (mostly in my code).
That is why I am explicitly initializing this directly in the definition.